modules/learn/pages/clusters-and-availability/rebalance.adoc

```diff
@@ -150,14 +151,35 @@ The rebalance process takes into account the nodes' CPU, RAM, and disk bandwidth
 By default, Couchbase Server rebuilds the indexes in their new locations during a rebalance.  
 If your cluster uses xref:learn:services-and-indexes/indexes/storage-modes.adoc#standard-index-storage[Standard Index Storage], you can choose to enable file-based rebalance which moves index files between nodes instead of rebuilding them. 
 Copying the index files is faster than having the target node rebuild the index from scratch.
-However, if you enable file-based rebalancing, you lose the ability to choose which Index Service node handles a specific index.  
+You cannot use file-based rebalance if you have enabled xref:learn:services-and-indexes/indexes/storage-modes.adoc#memory-optimized-index-storage[Memory Optimized Index Storage] because it does not store index data in files.
 
-NOTE: The first rebalance after you enable file-based index rebalance does not use the file-based rebalancing method. 
-Instead, the rebalance process rebuilds the indexes. 
-During this rebuild process, Couchbase Server adds metadata to the index files that enables later rebalances to move them between nodes. 
-After this initial rebalance, all later rebalances move the index files as long you as you keep file-base rebalance enabled.
+Couchbase Capella defaults to file-based index rebalance.
 
-If a file-base rebalance fails, you can try starting a new rebalance. 
+To learn how to enable file-based rebalance via the Couchbase Server Web Console, see File Transfer Based Rebalance under xref:manage:manage-settings/general-settings.adoc#index-storage-mode[Index Storage Mode]. To learn how to enable it via the REST API, see the `enableShardAffinity` parameter in xref:rest-api:post-settings-indexes.adoc[].
+
+// Remove after the 7.x series.
+ifeval::['{page-component-version}' == '7.6'] 
+NOTE: In Couchbase Server versions 7.6.0 and 7.6.1, when you enabled file-based rebalance you could not choose which Index Service nodes would contain an index.
+Couchbase Server 7.6.2 and later no longer have this restriction.
```

Comment:
> Couchbase Server 7.6.2 and later no longer have this restriction.

The restriction is removed only for CREATE INDEX command. ALTER INDEX command still goes with the same restrictions as 7.6.0 and 7.6.1